cartridges/int_algolia_sfra/cartridge/templates/default/slots/product/frequently-bought-together.isml

```diff
@@ -0,0 +1,17 @@
+<isscript>
+    productIds = [];
+    var productIdsString = "";
+
+    if (session.privacy.algoliaAnchorProducts) {
+        productIdsString = session.privacy.algoliaAnchorProducts;
+    } else {
+        for (var i = 0; i < slotcontent.content.length; i++) {
+            var product = slotcontent.content[i];
+            productIds.push(product.ID);
+        }
+        productIdsString = productIds.join(',');
+    }
+    delete session.privacy.algoliaProducts;
```

Comment:
Where is that defined? I don't see it, event in the demo repo

Reply:
It is a custom logic for passing `objectIDs` to recommendation widgets. Also, Salesforce Einstein using a similar approach via `context-object` but it is not possible to use it and we created a custom logic here via session variable. Also, mentioned about doc PR

Reply:
I understand now sorry.
It was a code before I created demo cartridge. Removed